src/mpyl/utilities/subprocess/__init__.py

```diff
@@ -21,6 +21,8 @@ def custom_check_output(
 
     command_argument = " ".join(command)
     logger.info(f"Executing: '{command_argument}'")
+    logger = logger.getChild("Subprocess")
```

Comment:
What does this do to what is printed in the logs? Will they they be interleaved?

Reply:
It doesn't seem to interleave it any more/less than with a print, at least it uses the logger now (with the nice formatting etc) and because of the child name it's a bit more clear what is happening.

The only downside is that is prints an extra info `INFO     mpyl.Subprocess  \[info]` that I can't seem to get rid off 😓